roles/ceph-monitor/tasks/main.yml

```diff
@@ -55,6 +55,20 @@
 - name: wait for client.admin key exists
   wait_for: path=/etc/ceph/ceph.client.admin.keyring
 
+- name: make sure ceph.log has proper permissions for logstash
+  file: dest=/var/log/ceph/ceph.log
+        state=touch
+        owner=root
+        group=adm
+        mode=0644
+
+- name: make sure ceph.audit.log has proper permissions for logstash
+  file: dest=/var/log/ceph/ceph.audit.log
+        state=touch
+        owner=root
+        group=adm
+        mode=0644
+
```

Comment:
If we are manually setting these to begin with, shouldn't we set the owner to ceph?